<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>

     <!-- 이름 -->
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; color: #333;</string>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>

     <!-- 아이디 -->
     <item>
      <widget class="QLabel" name="idLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; color: #333;</string>
       </property>
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>

     <!-- 비밀번호 -->
     <item>
      <widget class="QLabel" name="pwLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; color: #333;</string>
       </property>
       <property name="text">
        <string>비밀번호</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>

     <!-- 비밀번호 확인 -->
     <item>
      <widget class="QLabel" name="pwConfirmLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt; color: #333;</string>
       </property>
       <property name="text">
        <string>비밀번호 확인</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwConfirmLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>

     <!-- 회원가입 버튼 -->
     <item>
      <widget class="QPushButton" name="signupConfirmButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
         padding: 4px;
         background-color: #ff6f00;
         color: white;
         font-weight: bold;
         border-radius: 5px;
        </string>
       </property>
       <property name="text">
        <string>회원가입</string>
       </property>
      </widget>
     </item>

    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
